repository-data/webfiles/src/main/resources/site/freemarker/common/apimaster-article.ftl

```diff
@@ -81,6 +82,40 @@
         </div>
 
       </div>
-                          
+
 </article>
-                          
+
+<#-- Get section links in multiple elements -->
```

Comment:
I am not sure whether moving this function here is good idea. I know that this is only place where this function is used. However it was placed into imports.ftl as it is quite universal function and could be used in other places in the future.

If it was placed in imports.ftl it would be easier to find and reuse.

Reply:
Is this comment pointing at the correct source? Only GitHub has the comment block on apimaster-article.ftl

Reply:
It should be placed 1 line belo the comment. I meant about function 
<#function getNavLinksInMultiple sectionCompounds idprefix=''>

Reply:
I have moved that back now.